tests/genomic_file/test_genomic_file_resources.py

```diff
@@ -366,60 +303,7 @@ def exc():
     assert GenomicFile.query.count() == init + 1
 
 
-def test_filter_by_rg(client, indexd):
```

Comment:
why delete these tests?

Reply:
These tests are no longer needed since we've removed the functionality that it tests. Same rationale as above.